{"name": "test_m10", "status": "failed", "statusDetails": {"message": "AssertionError: assert False", "trace": "def test_m10():\n>      assert False\nE      assert False\n\ntest_Practice2.py:18: AssertionError"}, "start": 1747369327126, "stop": 1747369327126, "uuid": "891395ab-c5bb-41fd-9486-43f257fd4f4c", "historyId": "1a900fb1ca2791cb59d8e41085db3c68", "testCaseId": "1a900fb1ca2791cb59d8e41085db3c68", "fullName": "test_Practice2#test_m10", "labels": [{"name": "suite", "value": "test_Practice2"}, {"name": "host", "value": "Madhu"}, {"name": "thread", "value": "30016-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_Practice2"}]}